Monika After Story/game/script-stories.rpy

```diff
@@ -0,0 +1,196 @@
+# Module for Monika story telling
+#
+# Stories will get unlocked one at by session
+# The unlocking logic has been added to script-ch30
+# The topic that triggers the Story menu is monika_short_stories
+# Topic is unlocked at the beginning of the game and is not
+# random
+
+
+# dict of tples containing the stories event data
+default persistent.story_database = dict()
+
+
+# store containing stories-related things
+init -1 python in mas_stories:
+
+    # pane constants
+    STORY_X = 680
+    STORY_Y = 40
+    STORY_W = 450
+    STORY_H = 640
+    STORY_XALIGN = -0.05
+    STORY_AREA = (STORY_X, STORY_Y, STORY_W, STORY_H)
+    STORY_RETURN = "I changed my mind"
+    story_database = dict()
+
+
+# entry point for stories flow
+label mas_stories_start:
+    show monika at t21
+    python:
+        import store.mas_stories as mas_stories
+
+        # build menu list
+        stories_menu_items = [
+            (mas_stories.story_database[k].prompt, k, not seen_event(k), False)
+            for k in mas_stories.story_database
+            if mas_stories.story_database[k].unlocked
+        ]
+
+        # also sort this list
+        stories_menu_items.sort()
+
+        # final quit item
+        final_item = (mas_stories.STORY_RETURN, False, False, False, 20)
+
+    # call scrollable pane
+    call screen mas_gen_scrollable_menu(stories_menu_items, mas_stories.STORY_AREA, mas_stories.STORY_XALIGN, final_item=final_item)
+
+    # return value? then push
+    if _return:
+        $ pushEvent(_return)
+
+    show monika at t11
+    return _return

```

Comment:
I don't think you need to return anything here. Since you jumped from the topic version, this return gets passed back to `call_next_event`. To prevent any potential interference with quit returns, better not return anything here.